code/datums/gamemode/gamemode.dm

```diff
@@ -197,5 +197,9 @@
 	for(var/datum/faction/F in factions)
 		if(F.check_win())
 			return 1
+	if(emergency_shuttle.location==2 || station_was_nuked)
```

Comment:
change 2 to a define

Reply:
the shuttle controller doesn't even use those defines, dude

Reply:
Don't code for the codebase you have, code for the codebase you want

Reply:
What he's saying is that every time location is used in shuttle code should be reworked to use defines

Reply:
and the problem here is that it's inconsistent what it's checking for. Some check for CENTCOMM_Z to see if the shuttle's home, others just check for 1, and I don't want to take role datums out of scope

Reply:
I second this, keep this atomic